fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1656,21 +1656,17 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
     )
 
-    /**
-     * Indicates if juno onboarding feature is enabled.
-     */
-    val junoOnboardingEnabled: Boolean
-        get() = FxNimbus.features.junoOnboarding.value().enabled
-
     /**
      * Returns whether juno onboarding should be shown to the user.
+     *
+     * @param hasUserBeenOnboarded Boolean to indicate whether the user has been onboarded.
      * @param isLauncherIntent Boolean to indicate whether the app was launched on tapping on the
      * app icon.
      */
     fun shouldShowJunoOnboarding(hasUserBeenOnboarded: Boolean, isLauncherIntent: Boolean): Boolean {
         return if (!hasUserBeenOnboarded && isLauncherIntent) {
             FxNimbus.features.junoOnboarding.recordExposure()
```

Comment:
Wondering if we will still need this call? 

The plan is to land this change and uplift to Release since Juno will be our default onboarding experience. Currently, we have a rolling experiment that prefs ON the Juno Onboarding in Release. When we land this in Nightly and uplift this, we can follow up with Product to disable the Nimbus experiment that is keeping this on.

Reply:
Based on some of the info below, my take is probably not 🤔 however it may be useful to check in with the experiments team

    https://experimenter.info/jetstream/jetstream#enrollment-vs-exposure
    https://experimenter.info/telemetry#exposure
    https://experimenter.info/desktop/exposure